libs/langchain/langchain/chains/moderation.py

```diff
@@ -47,11 +47,15 @@ def validate_environment(cls, values: Dict) -> Dict:
             default="",
         )
         try:
+            from openai import OpenAI
             import openai
 
-            openai.api_key = openai_api_key
             if openai_organization:
-                openai.organization = openai_organization
+                # TODO: The 'openai.organization' option isn't read in the client API. You will need to pass it when you instantiate the client, e.g. 'OpenAI(organization=openai_organization)'
+                # openai.organization = openai_organization
+                client = OpenAI(organization=openai_organization, api_key=openai_api_key)
+            else:
+                client = OpenAI(api_key=openai_api_key)
             values["client"] = openai.Moderation
```

Comment:
I am not entirely sure how this line of code works and can't seem to find any documentation on it.
```
import openai
values["client"] = openai.Moderation
```
This seems to not be supported in v1.0.0 so I am currently looking for the alternative for v1.0.0

Reply:
I see now, this adds to the dictionary that is being returned.